glue.py

```diff
@@ -47,11 +49,19 @@ def options(self, module, ctx):
     def configure(self, module, ctx):
         ctx.load(self.waf_tools)
 
+def interpolate_string(string, env):
```

Comment:
This could be implemented through `string.Template(s).substitute(env)`, without messing with regexes.